tests/framework/vm_config.json

```diff
@@ -1,33 +1,34 @@
 {
-  "boot-source": {
-    "kernel_image_path": "vmlinux.bin",
-    "boot_args": "console=ttyS0 reboot=k panic=1 pci=off",
-    "initrd_path": null
-  },
-  "drives": [
-    {
-      "drive_id": "rootfs",
-      "path_on_host": "bionic.rootfs.ext4",
-      "is_root_device": true,
-      "partuuid": null,
-      "is_read_only": false,
-      "cache_type": "Unsafe",
-      "io_engine": "Sync",
-      "rate_limiter": null
-    }
-  ],
-  "machine-config": {
-    "vcpu_count": 2,
-    "mem_size_mib": 1024,
-    "smt": false,
-    "track_dirty_pages": false
-  },
-  "cpu-config": null,
-  "balloon": null,
-  "network-interfaces": [],
-  "vsock": null,
-  "logger": null,
-  "metrics": null,
-  "mmds-config": null,
-  "entropy": null
-}
+    "boot-source": {
```

Comment:
nit: the change seems to be just tabs so, do we actually need this?

Reply:
You are right, this change should not be here,